zazu/plugins/eslint_styler.py

```diff
@@ -0,0 +1,75 @@
+# -*- coding: utf-8 -*-
+"""ESLint plugin for zazu."""
+import zazu.styler
+zazu.util.lazy_import(locals(), [
+    'click',
+    'json',
+    'os',
+    'subprocess'
+])
+
+__author__ = "Patrick Moore"
+__copyright__ = "Copyright 2018"
+
+
+class ESLintStyler(zazu.styler.Styler):
+    """ESLint plugin for code styling."""
+
+    def style_string(self, string, filepath):
+        """Fix a string to be within style guidelines.
+
+        Args:
+            string (str): the string to style
+            filepath (str): the filepath of the file being styled
+
+        Returns:
+            Styled string.
+
+        """
+
+        eslint = 'eslint'
+        cwd = os.path.normpath(os.getcwd())
+        dirname = os.path.normpath(os.path.dirname(filepath))
+        loop_count = 0
+
+        local_eslint = os.path.join('node_modules', 'eslint', 'bin', 'eslint.js')
+
+        while True and loop_count < 100:
+            loop_count = loop_count + 1
+            maybe_eslint = os.path.join(dirname, local_eslint)
+
+            if os.path.isfile(maybe_eslint):
```

Comment:
@stopthatcow Still need to find a way to test this path

Reply:
You can mock os.path.isfile to return True in this case.